tests/flow/test_effects.py

```
@@ -0,0 +1,855 @@
+
+    def wait_for_effect(self):
+        self.wait_for_command('GRAPH.EFFECT')
+
+    def wait_for_query(self):
+        self.wait_for_command('GRAPH.QUERY')
+
+    def monitor_containt_effect(self):
+        for item in self.monitor:
+            if 'GRAPH.EFFECT' in item['command']:
+                return True
+        return False
```

Ruff (0.12.2)

Use `return any('GRAPH.EFFECT' in item['command'] for item in self.monitor)` instead of `for` loop

Replace with `return any('GRAPH.EFFECT' in item['command'] for item in self.monitor)`

(SIM110)